.github/workflows/codeql-codescan.yml

```
@@ -32,11 +32,11 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        language: [ 'typescript']
+        language: [ 'typescript' ]
```

YAMLlint (1.37.1)

[error] too many spaces inside brackets

(brackets)